fix: fixed wrong paths in import and export scripts

randompersonperson opened the pull request:
fixed the broken import and export scripts, also added a variable called $LIBRELINGO_ROOT_DIR to simplify some paths.

fixes issues #39 and #40

randompersonperson committed:
fix: fixed wrong paths in import and export scripts

fixed the broken import and export scripts, also added a variable called $LIBRELINGO_ROOT_DIR to simplify some paths.

giorginogreg commented:
Thank you so much for your work, first of all!☺️

Can you add please an example of the file for env vars and also some lines on the readme about how to use this file? (E.g. copy .env.example in .env ecc.)

Thanks!!

randompersonperson committed:
docs: added section to readme setup to initialise variables

giorginogreg commented:
There is the example file that is still missing, have you forget that, maybe? 

Please add it inside the folder mentioned, with an example KEY=value inside as the content, so that the people could then clone and change as needed. Thanks!! 😊

randompersonperson committed:
fix: added example vars.env file

randompersonperson commented:
I think that's everything

randompersonperson commented:
oh, also:
I'm confused whether npm or yarn is preferred for development usage, as the documentation on github pages says that npm is preferred, while nearly every example of doing anything is given with yarn.

randompersonperson committed:
fix: fixed problems with installAllExternalCourses

removed unnecessary arguments in package.json script; added notice in installExternalCourse.sh about vars.env

giorginogreg commented:
> oh, also: I'm confused whether npm or yarn is preferred for development usage, as the documentation on github pages says that npm is preferred, while nearly every example of doing anything is given with yarn.

It could be useful to transform everything in npm as default package manager, though it could be possible also to create a different set of scripts with yarn, to not exclude anyone.

randompersonperson commented:
Maybe we could put npm everywhere as a default, and also yarn as a secondary one? like having something like

### do a thing
`npm do something`
or `yarn do something`

giorginogreg commented:
Yeah it could be, but we should test all the commands to check if everything work.

randompersonperson commented:
of course

giorginogreg commented:
If you would like to contribute with this issue, you could proceed! Thank you SO much! :)

randompersonperson commented:
i will soon! i think i might submit a custom course downloader in a minute first though.
i saw #36 and couldn't resist making a script
